### DISCUSSION TITLE
Use of EEIO emission factor for trading company

### DISCUSSION THREAD

**Discussion by arby1005** [USEPA/supply-chain-factors#49] [Q&A]
Hi all,

I am calculating the emission for a trading company, so the purchase price of the product for sales is relatively low. Hence, we suspect we will underestimate the emission for the purchased goods. How will you suggest to do in order to better reflect the actual emission with the use of EEIO emission factor?

Thank you!

Arby

**Comment by bl-young** [comment]
Do you know the expected retail price? If so perhaps you could use that? The factors are in purchaser price.

**Reply by arby1005** [reply]
Hi bl-young,

Thank you for your response! May I clarify that purchaser price refers to the price the trading company procure the product, and retail price is the price they sell it to the customer? 

As you mention, the factors are in purchaser price, so I should be using the price the trading company procure the product for sale in calculation?

Thank you!

**Reply by bl-young** [reply]
Sorry no. I was suggesting that if the price paid by the trading company is generally below retail, than the retail price that is used to sell it to the customer may be closer to the purchaser price tracked in the model and could be a better proxy.
